### DISCUSSION TITLE
How to generate EF Core migration files?

### DISCUSSION THREAD

**Discussion by EthanZqh** [elsa-workflows/elsa-core#6120] [Design & tech]
I copied Elsa Entity Framework Core. MySQL project, and update MySqlDesignTimeVNet Factory class to use the MySQL version I want to use. How to generate EF Core migration files?
![migrations](https://github.com/user-attachments/assets/a8589966-b2ef-4658-890f-b5f42dfd17f3)

**Comment by elite-jose-porras** [comment]
@EthanZqh have you found any solution ?
cc: @sfmskywalker
